tests/examples/connext_dds/dynamic_data_nested_structs/java/DynamicDataNestedStruct.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Why do you need to add files for Java Dynamic data? Do they come from a merge?

Reply:
I added one test per language so they can be taken as a reference if someone wants to add another test.